File: totosuki/ABC/B/180.py
import math
N = int(input())
X = list(map(int, input().split()))
rslt1 = sum([abs(x) for x in X])
rslt2 = math.sqrt(sum([abs(x)**2 for x in X]))
rslt3 = max([abs(x) for x in X])
print(rslt1)
print(rslt2)
print(rslt3)

File: OCH_Tao/ABC/B/180.js
const X = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n")[1].split(" ").map(x => parseInt(x, 10));
console.log(X.map(Math.abs).reduce((a, b) => a + b));
console.log(Math.sqrt(X.map(x => Math.abs(x) ** 2).reduce((a, b) => a + b)));
console.log(Math.max(...X.map(Math.abs)));